### PR TITLE
German translation review

### PR DESCRIPTION
**Pull request by dianaolives**
@MWindel There are some images that are missing in both English and German guides. I am not sure if these should be removed.

### CODE REVIEW

File: content/benutzerhandbuch/cockpit-de-bundle/data-explorer.md [hunk 1]
```diff
@@ -217,6 +217,7 @@ Um diese Funktion nutzen zu können, muss zunächst ein Dashboard erstellt werde
 
 Klicken Sie auf **Mehr...** in der oberen Menüleiste und wählen Sie **Als Widget einem Bericht hinzufügen**, um ein Widget aus dem Daten-Explorer des Navigators zu erstellen.
 
+missing image?
```
Comment (dianaolives) [inline content/benutzerhandbuch/cockpit-de-bundle/data-explorer.md:220]: missing image

File: content/benutzerhandbuch/cockpit-de-bundle/exports.md [hunk 2]
```diff
@@ -64,6 +64,7 @@ Neben dem Filtern nach bestimmten Objekten oder Zeiträumen können Daten nach b
 
 Aktivieren bzw. deaktivieren Sie diese Felder mit dem Umschalter.
 
+missing image?
```
Comment (dianaolives) [inline content/benutzerhandbuch/cockpit-de-bundle/exports.md:67]: missing image

File: content/benutzerhandbuch/cockpit-de-bundle/smart-rules.md [hunk 3]
```diff
@@ -138,8 +138,12 @@ Führen Sie folgende Schritte aus, um eine Schwellenwertregel zu definieren:
 1. Navigieren Sie im Menü Gruppen zu dem Objekt (Gruppe oder Gerät), auf welches Sie den Schwellenwert anwenden möchten.
 2. Wechseln Sie zur Registerkarte **Daten-Explorer**.
 3. Wenn der Datenpunkt, der den Schwellenwert festlegen soll, standardmäßig nicht sichtbar ist, wählen Sie **Datenpunkt hinzufügen** und [fügen Sie einen Datenpunkt hinzu](#add-data-points).
-4. Öffnen Sie über das Menüsymbol das Kontextmenü für den entsprechenden Datenpunkt und klicken Sie auf **Smart Rule erstellen**. <br><br> <img src="/images/benutzerhandbuch/cockpit/cockpit-smart-rules-data-point.png" name="Data point example"/>
+4. Öffnen Sie über das Menüsymbol das Kontextmenü für den entsprechenden Datenpunkt und klicken Sie auf **Smart Rule erstellen**.
+
+missing image?
```
Comment (dianaolives) [inline content/benutzerhandbuch/cockpit-de-bundle/smart-rules.md:143]: missing image

File: content/benutzerhandbuch/cockpit-de-bundle/using-widgets.md [hunk 4]
```diff
@@ -100,6 +100,7 @@ Auf der Registerkarte **Konfiguration** des Widget-Editors können Sie das Vollt
 
 Über das Volltextsuche-Feld können Sie Assets in der gesamten Hierarchie finden, doch dies erfordert exakte Übereinstimmungen, z. B. den gesamten Namen eines Assets.
 
+missing image?
```
Comment (dianaolives) [inline content/benutzerhandbuch/cockpit-de-bundle/using-widgets.md:103]: missing image

File: content/benutzerhandbuch/device-management-de-bundle/device-details.md [hunk 5]
```diff
@@ -371,6 +371,7 @@ Beispiel:
 
 In der Registerkarte **Netzwerk** können Parameter für das mobile Netzwerk (WAN) und das lokale Netzwerk (LAN) angezeigt und konfiguriert werden.
 
+english image?
```
Comment (dianaolives) [inline content/benutzerhandbuch/device-management-de-bundle/device-details.md:374]: Should there be a screenshot in German here?

File: content/benutzerhandbuch/cockpit-de-bundle/smart-rules.md (hunk 3 above)
Comment (MWindel) [reply content/benutzerhandbuch/cockpit-de-bundle/smart-rules.md:143]: This screenshot is missing in the English documentation as well. @BeateRixen, was this removed?

File: content/benutzerhandbuch/cockpit-de-bundle/using-widgets.md (hunk 4 above)
Comment (MWindel) [reply content/benutzerhandbuch/cockpit-de-bundle/using-widgets.md:103]: This screenshot is missing in the English documentation as well. @BeateRixen, was this removed?

File: content/benutzerhandbuch/device-management-de-bundle/device-details.md (hunk 5 above)
Comment (dianaolives) [reply content/benutzerhandbuch/device-management-de-bundle/device-details.md]: German screenshot has been added

File: content/benutzerhandbuch/cockpit-de-bundle/data-explorer.md (hunk 1 above)
Comment (dianaolives) [reply content/benutzerhandbuch/cockpit-de-bundle/data-explorer.md]: Removed

File: content/benutzerhandbuch/cockpit-de-bundle/exports.md (hunk 2 above)
Comment (dianaolives) [reply content/benutzerhandbuch/cockpit-de-bundle/exports.md:67]: Contacted Jan for this screenshot in English and German

File: content/benutzerhandbuch/cockpit-de-bundle/smart-rules.md (hunk 3 above)
Comment (dianaolives) [reply content/benutzerhandbuch/cockpit-de-bundle/smart-rules.md]: Line removed

File: content/benutzerhandbuch/cockpit-de-bundle/exports.md (hunk 2 above)
Comment (dianaolives) [reply content/benutzerhandbuch/cockpit-de-bundle/exports.md]: Done.

**Review by MWindel — APPROVED** [review]

**Pull request merged**